### PR TITLE
dist.xml should contain all packages

### PR DESCRIPTION
**Pull request by alexandraorth**

### CODE REVIEW

File: grakn-dist/src/dist.xml [hunk 1]
```diff
@@ -42,6 +42,8 @@
                 <include>pom.xml</include>
                 <include>LICENSE.txt</include>
                 <include>grakn-core/**</include>
+                <include>grakn-client/**</include>
+                <include>grakn-test-profiles/**</include>
```
Comment (aelred) [inline grakn-dist/src/dist.xml]: Does it matter if `grakn-test-profiles` is here?

**Comment by alexandraorth** [comment]
please retest - `ai.grakn.test.graql.reasoner.inference.GenealogyTest.testGrandParentship`

**Comment by alexandraorth** [comment]
please retest - `ai.grakn.test.graql.reasoner.inference.GenealogyTest.testGrandParentship`

**Comment by alexandraorth** [comment]
please retest this please - `ai.grakn.test.graql.reasoner.inference.GenealogyTest.testGrandParentship`

**Comment by alexandraorth** [comment]
please retest this please

**Pull request merged**